app/dashboard/templates/dashboard/index-vue.html

```diff
@@ -956,16 +913,18 @@ <h6 class="font-weight-bold mb-3">Invite [[numUsers]] Users to the Bounty</h6>
     <script src='{% static "v2/js/vue-components.js" %}'></script>
     {{orgs|json_script:"sponsor-list"}}
     {{hackathon_obj|json_script:"hackathon-object"}}
+    {{is_registered|json_script:"hackathon-registration"}}
```

Comment:
ohh ok so this will get into <script id=hackathon-registration> ?
But why changeing into it instead of json.dumps(True if is_registered else False)?

Reply:
another option also if you did it because you are using is_registered in some other django html view we and need both True and true for js then we can just do `{{ document.is_registered = is_registered|yesno:"true,false" }};`